### PR TITLE
[P4-2908] Create new identity bar component

### PR DESCRIPTION
**Pull request by teneightfive**
### What changed

This creates a new component to display the summary information for
a move or person page.

[**COMPONENT EXAMPLE**](https://booksecuremove-pr-1832.herokuapp.com/components/identity-bar)

### Why did it change

This component will be used by the new layouts for the move and person pages.

### Issue tracking

- [P4-2908](https://dsdmoj.atlassian.net/browse/P4-2908)

## Screenshots

<kbd>image</kbd>

<kbd>image</kbd>

#### Automated testing

- [x] Added unit tests to cover changes
- [ ] Added end-to-end tests to cover any journey changes

#### Manual testing

Has been tested in the following browsers:

- [x] Chrome
- [x] Firefox
- [ ] Edge
- [ ] Internet Explorer

### Environment variables

- [x] No environment variables were added or changed

### CODE REVIEW

**Review by Thource — APPROVED** [review]

**Review by thomasleese — APPROVED** [review]

File: common/components/identity-bar/_identity-bar.scss [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+  .is-sticky & {
+    @include govuk-font($size: 19, $weight: bold);
+    display: inline;
+  }
+}
+
+.app-identity-bar__summary {
+  @include govuk-font($size: 19);
+  margin: govuk-spacing(3) 0 0;
```
Comment (teneightfive) [inline common/components/identity-bar/_identity-bar.scss:43]:
```suggestion
  margin: govuk-spacing(1) 0 0;
```

File: common/components/identity-bar/_identity-bar.scss [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+  .is-sticky & {
+    @include govuk-font($size: 16);
+    margin: govuk-spacing(1) 0 0;
```
Comment (teneightfive) [inline common/components/identity-bar/_identity-bar.scss:47]:
```suggestion
```

File: common/components/identity-bar/_identity-bar.scss [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+.app-identity-bar__actions {
+  margin: 0;
+  padding: govuk-spacing(6) 0 0;
+  list-style: none;
+
+  @include mq($from: desktop) {
+    float: right;
+  }
+
+  .is-sticky & {
```
Comment (teneightfive) [inline common/components/identity-bar/_identity-bar.scss:60]:
```suggestion
  .govuk-button {
    margin-bottom: 0;
  }
  
  .is-sticky & {
```

**Comment by ElizabethGoddard** [comment]
Hey @teneightfive, I think you said you are still working on vertically aligning the call to action to the centre which would be great but other wise this is fab! Thank you!

**Review by ElizabethGoddard — COMMENTED** [review]
Aside from vertically aligning the call to action to the centre this looks complete (testing to follow). Thank you Dom!

**Pull request merged**